### ISSUE TITLE
OpenCFP Central asking for http authentication

### ISSUE THREAD

**Issue by tdutrion** [opencfp/opencfp#1211]
# Steps required to reproduce the problem

1. Create an app in opencfp central
2. Add the id / pass in config/production.yml
3. Clear cache
4. Try to connect

# Expected Result

* Redirect on OpenCFP Central, and then to the opencfp instance

# Actual Result

* OpenCFP central is asking for an http basic auth

<img width="1112" alt="screenshot 2019-01-31 at 02 05 19" src="https://user-images.githubusercontent.com/1712222/52023325-f6a6dd00-24fc-11e9-973c-780e3ca8e210.png">

**Comment by chartjes** [comment]
@tdutrion I'm working on debugging what is going on here. Locally I have it doing the complete cycle but I am getting a different error.

**Comment by chartjes** [comment]
Okay, so here is what I did:

1. Logged into OpenCFP Central
2. Created a new OAuth client
3. Made sure my local copy of OpenCFP's configuration file had the values for client ID, secret key, and redirect URL

Here is the relevant section of my configuration file, with my copy of OpenCFP running locally on my laptop.

opencfpcentral:
  sso: on
  clientId: 52
  clientSecret: <redacted>
  authorizeUrl: https://www.opencfpcentral.com/oauth/authorize?
  redirectUri: http://localhost:8080/sso/redirect
  resourceUri: https://www.opencfpcentral.com/api/user
  tokenUrl: https://www.opencfpcentral.com/oauth/token


4. Tried logging in via OpenCFP Central
5. Successfully ended up back on OpenCFP with a new account having been created 

In cases where I got the HTTP Auth request, I fixed it by deleting my OAuth client and creating a new one.

I realize that this is not the best solution but until I can dig in and debug things on the OpenCFP Central side, this is a working solution.
